doc: Start API Only

**Problem**
the elektron part fails to build on my machine, thats frustrating because i dont need the frontend. i only want to use the api. but i cant start it. 

**Success Criteria**
a command for example "make dev-api" that makes the api run

**Additional context**
i am running nixos and it doesnt allow dynamiclly linkt executables by default so make dev fails with 
```
yarn run v1.22.19
$ concurrently --kill-others "yarn dev:web" "wait-on http://localhost:3000 && yarn dev:electron"
$ yarn workspace jan-web dev
$ next dev
[0]    ▲ Next.js 14.0.1
[0]    - Local:        http://localhost:3000
[0] 
[0]  ✓ Ready in 1912ms
[0]  ○ Compiling /page ...
[0]  ✓ Compiled /page in 9.3s (5618 modules)
$ yarn copy:assets && yarn workspace jan dev
$ cpx "models/**" "electron/models/" && cpx "pre-install/*.tgz" "electron/pre-install/" && cpx "docs/openapi/**" "electron/docs/openapi"
$ yarn copy:assets && tsc -p . && electron .
$ rimraf --glob "./pre-install/*.tgz" && cpx "../pre-install/*.tgz" "./pre-install"
[1] Could not start dynamically linked executable: /home/leonard/Projekte/ai/jan/node_modules/electron/dist/electron
[1] NixOS cannot run dynamically linked executables intended for generic
[1] linux environments out of the box. For more information, see:
[1] https://nix.dev/permalink/stub-ld
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
[1] Exit code: 127
[1] Command: /nix/store/vz13mi0w75q96sfjxz2ylnv8812hvf34-nodejs-20.11.1/bin/node
[1] Arguments: /nix/store/s5lkaji89kp00dxi6qj782a2hdb5p6lp-yarn-1.22.19/libexec/yarn/lib/cli.js dev
[1] Directory: /home/leonard/Projekte/ai/jan/electron
[1] Output:
[1] 
info Visit https://yarnpkg.com/en/docs/cli/workspace for documentation about this command.
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[1] wait-on http://localhost:3000 && yarn dev:electron exited with code 127
--> Sending SIGTERM to other processes..
[0] 
[0] yarn dev:web exited with code 1
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
make: *** [Makefile:33: dev] Fehler 1

[leonard@laptop:~/Projekte/ai/jan]$ 
```

--- a/README.md
+++ b/README.md
@@ -209,6 +209,12 @@ Contributions are welcome! Please read the [CONTRIBUTING.md](CONTRIBUTING.md) fi
 
 This will start the development server and open the desktop app.
 
+3. (Optional) **Run the API server without frontend**
+
+   ```bash
+   yarn dev:server
+   ```
+
 ### For production build
 
 ```bash

